Common/Transforms/itkGridScheduleComputer.hxx

```diff
@@ -40,7 +40,7 @@ GridScheduleComputer<TTransformScalarType, VImageDimension>::GridScheduleCompute
 
   m_ImageOrigin.Fill(0.0);
   m_ImageSpacing.Fill(1.0);
-  m_ImageDirection.Fill(0.0);
+  m_ImageDirection.SetIdentity();
```

Comment:
@mstaring @stefanklein Could this be a serious bug? (Zero-filling looks like an incorrect way to initialize an direction matrix.) The fix is easy, anyway, as you see 😃

Reply:
Probably line 161 in that file saves us, right?

Reply:
I see. Ok, then I guess we were saved by:
https://github.com/SuperElastix/elastix/blob/030a80d19cd3c68ad46481b438170ad946917c61/Components/Transforms/AdvancedBSplineTransform/elxAdvancedBSplineTransform.hxx#L165

But a good catch anyway!

Reply:
Thanks @mstaring !  I just added the following to the commit text:

> Note that this bug never affected any actual use case of elastix, because the ImageDirection of a GridScheduleComputer is in practice always set, in the PreComputeGridInformation() member function of the transform that uses the GridScheduleComputer. As remarked by Marius Staring.